### ISSUE TITLE
[Accessibility]: Code monitor edit page isn't great looking on mobile

### ISSUE THREAD

**Issue by benvenker** [sourcegraph/sourcegraph#35152]
### Audit type

Mobile device navigation

### Problem description

On mobile, the edit code monitor UI has a very wide margin relative to what's necessary, which causes things like the query display to be very difficult to read.

### Expected behavior

I'd expect the content to take up more width and for the code monitor query to be readable

### Additional details

![image](https://user-images.githubusercontent.com/4605817/167484659-9129a7e2-252f-42bf-8e11-33f905b9ee66.png)

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by limitedmage** [comment]
Duplicate of #35733
